src/components/HighlightCard/HighlightCard.test.jsx

```diff
@@ -0,0 +1,77 @@
+import { render, screen } from "@testing-library/react";
+import HighlightCard from "./HighlightCard";
+import { describe, it, expect } from "vitest";
+
+describe("<HighlightCard />", () => {
+  it("renders the tile prop", () => {
+    render(
```

Comment:
Note that you repeat same stuff few times, you can:

- either create a method that calls the render function
- create the expects in the same it (since you're testing first the same type of render)